portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -68,6 +71,22 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     // Respond with the result.
     response.setContentType("application/json;");
     response.sendRedirect("index.html#contact");
+
+    // Sentiment Analysis
+    String message = request.getParameter("comment-input");
+
+    Document doc =
+        Document.newBuilder().setContent(message).setType(Document.Type.PLAIN_TEXT).build();
+    LanguageServiceClient languageService = LanguageServiceClient.create();
+    Sentiment sentiment = languageService.analyzeSentiment(doc).getDocumentSentiment();
+    float score = sentiment.getScore();
+    languageService.close();
+
+    System.out.println(score);
```

Comment:
is this needed for production? if not we can remove it :D if needed, we can add a clearer logging message like "Sentiment score of <comment> is <score>"

Reply:
okay! Its for debugging purposes, I will remove it after completing the whole set up